src/protagonist/Portal/Pages/Exception.cshtml

```diff
@@ -0,0 +1,26 @@
+﻿@page
+@model ExceptionModel
+@{
+    ViewData["Title"] = "Error";
+}
+
+<h1 class="text-danger">Error.</h1>
+<h2 class="text-danger">An error occurred while processing your request.</h2>
+
+@if (Model.ShowRequestId)
+{
+    <p>
+        <strong>Request ID:</strong> <code>@Model.RequestId</code>
+    </p>
+}
+
+<h3>Development Mode</h3>
+<p>
+    Swapping to the <strong>Development</strong> environment displays detailed information about the error that occurred.
```

Comment:
It's been a while since I've done anything with custom error handling in razor but I don't think there's a need to copy the markup of the default error page - there is a middleware to handle that. It will enable full stack trace etc to be shown. 

As is would `Swapping to the <strong>Development</strong> environment displays detailed information about the error that occurred.` happen?